test/cases/block_quote.test

```diff
@@ -0,0 +1,9 @@
+```
+> Basic
+> block quote.
+.
+<blockquote>
+<p>Basic
+block quote.</p>
+</blockquote>
+```
```

Comment:
Could you move the rest of the tests from https://github.com/lpil/jot/blob/main/test/cases_unimplemented/block_quote.test please? Thank you.

Reply:
I only moved the one test over, because it's the only one it can pass yet, at least until parse_inner can deal with things like <em> <strong> and the like, if I move the other tests over they will fail.

Reply:
It needs to have at least tests for parsing the nested elements that can be parsed at the top today such as headings and links.